### PR TITLE
params: set TerminalTotalDifficultyPassed to true

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
We passed the terminal total difficulty on block 
```
> eth.getBlock(15537394).hash
"0x56a9bb0302da44b8c0b3df540781424684c3af04d0b7a38d72842b762076a664"
```

### CODE REVIEW

File: params/config.go [hunk 1]
```diff
@@ -59,25 +59,26 @@ var (
...
+		ByzantiumBlock:                big.NewInt(4_370_000),
+		ConstantinopleBlock:           big.NewInt(7_280_000),
+		PetersburgBlock:               big.NewInt(7_280_000),
+		IstanbulBlock:                 big.NewInt(9_069_000),
+		MuirGlacierBlock:              big.NewInt(9_200_000),
+		BerlinBlock:                   big.NewInt(12_244_000),
+		LondonBlock:                   big.NewInt(12_965_000),
+		ArrowGlacierBlock:             big.NewInt(13_773_000),
+		GrayGlacierBlock:              big.NewInt(15_050_000),
+		TerminalTotalDifficulty:       MainnetTerminalTotalDifficulty, // 58_750_000_000_000_000_000"0x56a9bb0302da44b8c0b3df540781424684c3af04d0b7a38d72842b762076a664"_000
```
Comment (fjl) [inline params/config.go:79]: Not sure what happened to this comment.

Comment (holiman) [inline params/config.go:79]:
```suggestion
		TerminalTotalDifficulty:       MainnetTerminalTotalDifficulty, // 58_750_000_000_000_000_000_000
```

**Review by holiman — APPROVED** [review]
LGTM

**Comment by MariusVanDerWijden** [comment]
My stupid os sometimes pastes the buffer when I click

**Comment by sandakersmann** [comment]
We passed the terminal total difficulty on the block before that: https://etherscan.io/block/15537393

0x55b11b918355b1ef9c5db810302ebad0bf2544255b530cdce90674d5887bb286

Block 15537394 was the first PoS block.

**Pull request merged**